### PR TITLE
OPS-1690 add command for single check

### PR DESCRIPTION
**Pull request by missmaggiemo**
Add Sensu check command to alert on a single Pingdom check. We'd like to have a record of certain individual Sensu checks being "up" or "down", independent of other checks.

### CODE REVIEW

**Comment by c4urself** [comment]
cool stuff, how do you figure out the `check_id`?

File: bin/check-pingdom-single.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+#
+# Check Pingdom single alert (checks down)
+# ===
+#
+# Alerts if too many websites are down in the Pingdom account.
```
Comment (c4urself) [inline bin/check-pingdom-single.rb]: probably want to update this part

**Comment by missmaggiemo** [comment]
Easily discoverable in the URL for the report or editing the check
```
https://my.pingdom.com/reports/uptime#check=[CHECK_ID]&daterange=7days&tab=uptime_tab
```
or
```
https://my.pingdom.com/newchecks/checks#check=[CHECK_ID]
```
Just added to the check file. :)

File: bin/check-pingdom-single.rb (hunk 1 above)
Comment (missmaggiemo) [inline bin/check-pingdom-single.rb]: 😳 😅

File: bin/check-pingdom-single.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+         required: true
+  option :application_key,
+         short: '-k APP_KEY',
+         long: '--application-key APP_KEY',
+         required: true
+  option :check_id,
+         short: '-i CHECK_ID',
+         long: '--check-id CHECK_ID',
+         required: true
+
```
Comment (mdelaney) [inline bin/check-pingdom-single.rb]: nit - unneeded blank line

Comment (missmaggiemo) [inline bin/check-pingdom-single.rb]: Removed!

**Pull request merged**